src/morse/sensors/camera.py

```diff
@@ -83,7 +83,8 @@ def __init__(self, obj, parent=None):
         the process potentially slower and may introduce some glitches
         if some objects are not properly synchronised).
         """
-        self._offscreen_create = getattr(blenderapi.render(), 'offScreenCreate', None)
+        #self._offscreen_create = getattr(blenderapi.render(), 'offScreenCreate', None)
```

Comment:
Add a comment explaining the situation, or cleanup commented code.

Reply:
As a reminder FBO prenvent from retrieving Z-buffer, this change reverts the "one scene per camera resolution" feature.